### PR TITLE
HDI-ADL changes

### PR DESCRIPTION
**Pull request by yoreddy**
can someone please take a look at this and merge? @shefaliv

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@yoreddy__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by stankovski** [comment]
@azuresdkci add to whitelist

File: src/ResourceManagement/HDInsight/HDInsight.Tests/ScenarioTests/DataLakeTests.cs [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                var resourceGroup = HDInsightManagementTestUtilities.CreateResourceGroup(resourceManagementClient);
+
+                //set variables
+                const string dnsname = "hdisdk-datalake5";
+
+                var spec = GetDataLakeClusterParametersExtended();
+
+                var createresponse = client.Clusters.Create(resourceGroup, dnsname, spec);
+
+                //TODO: Assert if data lake configurations are set once shefali adds GetConfigurations support
```
Comment (shefaliv) [inline src/ResourceManagement/HDInsight/HDInsight.Tests/ScenarioTests/DataLakeTests.cs]: I believe Nemanja has added this support.

**Comment by shefaliv** [comment]
approved.

**Pull request merged**